operations/app/src/modules/key_vault/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_version = ">= 0.14"
+}
+
+locals {
+  object_ids = ["34232fe8-00ad-4bd0-9afb-eb9b3cc93ffe",
```

Comment:
These are developer ids, correct? Might be good to comment who they are, and also provide some clarity for a future reader we're not exposing a secret or something in our config.

Reply:
Good call.  I have made comments that these are developer IDs and identified what each one is.